scene/gui/dialogs.cpp

```diff
@@ -195,6 +195,14 @@ String AcceptDialog::get_ok_button_text() const {
 	return ok_button->get_text();
 }
 
+void AcceptDialog::set_ok_button_tooltip_text(String p_ok_button_tooltip_text) {
+	ok_button->set_tooltip_text(p_ok_button_tooltip_text);
+}
+
+String AcceptDialog::get_ok_button_tooltip_text() const {
+	return ok_button->get_tooltip_text();
+}
+
```

Comment:
Raising eyebrows, these new methods feel a little _too_ niche. Should not exist unless other things in the engine would greatly benefit from them.

Reply:
> get_ok_button() already exists, so these methods are not needed.

ah nice